draft-ietf-lake-reqs.md

```diff
@@ -186,24 +185,26 @@ Moreover, it shall be possible for the receiving endpoint to detect a replayed A
 
 Furthermore, the endpoints shall be able to verify that the identity of the other endpoint is an acceptable identity that it is intended to authenticate to. (This requirement extends beyond the AKE in that the application must enable access to information about acceptable identities without compromising the overall lightweightness of the AKE.)
 
+## Confidentiality {#confidentiality}
```

Comment:
The document missed a "Confidentiality" section; the confidentiality goals were earlier misplaced within Crypto Agility